easybuild/easyconfigs/s/Seqmagick/Seqmagick-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,38 @@
+##
```

Comment:
@ysagon duplicate easyconfig with wrong name, this one should probably just removed?